robots.txt

```diff
@@ -0,0 +1,4 @@
+User-agent: *
```

Comment:
This file may not be needed since these pages won't be reachable by an unauthenticated crawler anyway, but I figure it is worth including until I read more and make sure

Reply:
After my other one is merged we should add `/blendPro/success`

Reply:
Yeah, but I need to do a bit of reading first, I think it may be a moot point if the robot cannot reach the page by following links